src/Display/Modal/Modal.test.tsx

```diff
@@ -168,3 +168,44 @@ it('should center the Modal vertically when centering is true', () => {
   const modalContainer = getByTestId('modal-container');
   expect(modalContainer).toHaveStyle('align-items: center;');
 });
+
+test('<Modal> should be visible when isVisible changes from false to true', async () => {
+  const ModalComponent = ({ isVisible }: { isVisible: boolean }) => (
+    <Modal
+      title={props.title}
+      isVisible={isVisible}
+      footer={[<button key="button">{props.footer}</button>]}
+      onClose={props.onClose}
+    >
+      <p>{props.content}</p>
+    </Modal>
+  );
+
+  const { queryByTestId, rerender } = render(
+    <ModalComponent isVisible={false} />
+  );
+  rerender(<ModalComponent isVisible={true} />);
```

Comment:
Should check `expect(queryByTestId('modal-container')).not.toBeInTheDocument();` before this line?

Reply:
the test description is `<Modal> should be visible when isVisible changes from false to true`, so I didn't check it, and I think there is already a test checked it. What do you think?

Reply:
but I can add